src/routes/userapp/details-page/loader.tsx

```diff
@@ -3,12 +3,21 @@ import { defer, LoaderFunctionArgs } from "react-router-dom";
 import { Item } from "../../../types";
 import { CommentList, DetailsPageConfig } from "../types";
 import { BACKEND_URL } from "../../../query";
+import { getAccessToken } from "../../../auth/utils";
 
 const fetchDetailsConfig = async (
   storeId: string,
 ): Promise<DetailsPageConfig> => {
+  const token = getAccessToken();
   const res = await fetch(
-    `${BACKEND_URL}/store-configs/${storeId}/detailsPageConfig`,
+    // toggled, as backend do not send attributes in detailsPageCongig, as expected
```

Comment:
Ja  bym w ogole zrezygnował z uderzania na details-page-config (i co za tym idzie wywaliłbym hooka useDetailsPageConfig) i wszedzie uzywałym bym useStoreConfig. To juz jest zaimplementowane, wystarczy wszedzie zastąpić useDetailsPageConfig przez useStoreConfig (zaimportowanego z folderu wrapper, bo ogolnie są 3 hooki o tej samej nazwie xd)